_data/mentors.yml

```diff
@@ -266,3 +266,12 @@
   image: Ayush.jpeg
   lat: 25.4275
   lng: 81.7728
+  
+  
+- name: Deepjyoti Mondal
+  github: djmgit
+  twitter: djm_dev96 
+  image: Mentor.jpeg
```

Comment:
Good going, please can you rename the file to something more sensible? By his name would be preferable.

Reply:
ohkk

Reply:
Thanks, and good work.

Reply:
image still says - `Mentor.jpeg`